Source/UI/Controllers/CardViewController.swift

```diff
@@ -181,8 +181,8 @@ public class CardViewController: UIViewController,
         cardView.cardNumberInputView.hideError()
         cardView.expirationDateInputView.hideError()
         cardView.cvvInputView.hideError()
-
-        let cardValidator = CardValidator()
+        // Potential Task: do not hardcode environment
+        let cardValidator = CardValidator(environment: .sandbox)
```

Comment:
@harry-brown-cko how are we planning to get the environment passed here. I dont think its exposed from CheckoutAPIservice.